grow/translators/google_sheets.py

```diff
@@ -48,6 +48,7 @@ class GoogleSheetsTranslator(base.Translator):
     }
     KIND = 'google_sheets'
     HEADER_ROW_COUNT = 1
+    HEADER_LABELS = ['', '', 'Location']
```

Comment:
Maybe add a comment here for "# Source locale, translation locale, message location."